imports/server/publications/pendingGuessesForSelf.ts

```diff
@@ -49,7 +49,7 @@ class PendingGuessWatcher {
 
         Object.entries(roles).forEach(([huntId, huntRoles]) => {
           if (huntId === GLOBAL_SCOPE || !huntRoles.includes('operator')) return;
-          this.huntGuessWatchers[huntId] ||= new JoinPublisher(this.sub, huntGuessSpec, { state: 'pending', hunt: huntId });
+          this.huntGuessWatchers[huntId] ||= publishJoinedQuery(this.sub, huntGuessSpec, { state: 'pending', hunt: huntId });
```

Comment:
Hmmm, this might be a little pedantic, and I don't think it will actually negatively impact anything currently, but this doesn't seem quite right -- this is going to mark the whole `pendingGuessesForSelf` sub as `ready()` as soon as a single `Hunt` worth of guess watchers has run, which is too soon.

I think it'd be more consistent for readiness to be signaled at the top-level of the subscription -- it's a single bit that's associated with the subscription as a whole, and the top-level is the only place where there's sufficient context to know if this is the only `publishJoinedQuery` that will be run.

Reply:
Mmm yeah, I think you're right. This should actually be using `PublicationMerger`